UI/LinkText.cpp

```diff
@@ -571,6 +575,15 @@ void TextLinker::MarkLinks() {
     SetLinkedText(std::move(marked_text));
 }
 
+std::string LinkStringIfPossible(std::string raw, std::string user_string) {
+    if      (auto x = GetBuildingType(raw)) return LinkTaggedPresetText(VarText::BUILDING_TYPE_TAG, raw, user_string);
+    else if (auto x = GetSpecies(raw))      return LinkTaggedPresetText(VarText::SPECIES_TAG,       raw, user_string);
+    else if (auto x = GetSpecial(raw))      return LinkTaggedPresetText(VarText::SPECIAL_TAG,       raw, user_string);
+    else if (auto x = GetShipHull(raw))     return LinkTaggedPresetText(VarText::SHIP_HULL_TAG,     raw, user_string);
+    else if (auto x = GetShipPart(raw))     return LinkTaggedPresetText(VarText::SHIP_PART_TAG,     raw, user_string);
+    else return user_string;
```

Comment:
focusses are still missing

Reply:
and policies are missing.

Reply:
> > > please dont add another species to this PR if you want to be merged.

if its all the same, i'd like to leave them in and finish their icon and add some more text to them and i promise i wont do any more species in the same pr after that.

> 
> you can always add them in another PR. one thing we should address about them before adding: the sly fluff sais one reason for them going unnoticed is that all experts agree that life on gas giants can not evolve. gameplay wise, I would like them (maybe even all gaseous species to come) to be also stealthy, so one is not able to distinguish a Sly from a Thenian colony before one has enough detection strength.

i believe i have addressed all those concerns – they are now stealthy, the sly's description is a bit more accomodating to the idea of other gaseous life out there to be found if one has the right tech to find them.

and the pedia texts are a little more readable by applying the same style of linked list formatting in all the places, so that is e.g.: "Link One,  Comma two spaces,  Link two,  and so on" with the two spaces instead of one so as to better visually distinguish where the boundaries of the multiword list entries are, and all linked up and clickable, and nicely comma separated.

Reply:
`VarText` is arguably a part of the UI, though it's really sort of glue between the UI and game logic code, so I don't mind that the UI knows about the various TAG strings. More annoying is that this code has to delcare / use all the content getters, but I don't have a better solution or place for this...

I don't think there's an existing "look up this string until you find something named that in the content managers" function. If implementing that, the [ContentType](https://github.com/freeorion/freeorion/blob/master/universe/Conditions.h#L44) enum that could maybe be used as the return value.